BlockchainSdk/ExternalLinkProvider/NEARExternalLinkProvider.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NEARExternalLinkProvider.swift
+//  BlockchainSdk
+//
+//  Created by Andrey Fedorov on 12.10.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+
+struct NEARExternalLinkProvider: ExternalLinkProvider {
+    var testnetFaucetURL: URL? {
+        return URL(string: "https://near-faucet.io/")
+    }
+
+    private let isTestnet: Bool
+
+    init(
+        isTestnet: Bool
+    ) {
```

Comment:
Как будто лишнее тоже)
```suggestion
    init(isTestnet: Bool) {
```

Reply:
Поправил тут и еще в месте, где передавался единственный параметр в конструктор